src/Microsoft.DotNet.XHarness.CLI/Microsoft.DotNet.XHarness.CLI.csproj

```diff
@@ -1,5 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <!-- When updating, avoid using 'latest' url as this can make the same commit build differently on different days -->
+    <WindowsAndroidSdkUrl>https://dl.google.com/android/repository/platform-tools_r29.0.6-windows.zip</WindowsAndroidSdkUrl>
+    <WindowsAndroidSdkFileName>windows-android-platform-tools.zip</WindowsAndroidSdkFileName>
+  
+    <!-- TODO: Figure out minimal set of binaries for linux, then copy them.  Until then this is unsued. -->
+    <LinuxAndroidSdkUrl>https://dl.google.com/android/repository/platform-tools_r29.0.6-linux.zip</LinuxAndroidSdkUrl>
+    <LinuxAndroidSdkFileName>linux-android-platform-tools.zip</LinuxAndroidSdkFileName>
+  </PropertyGroup>
+
+  <Target Name="DownloadAndroidSdks" BeforeTargets="Build" >
```

Comment:
Just asking because I don't know MSBuild much - will this download on every Build whenever I want to run in VS or how does it know it shouldn't run this target?

Or should there be a condition `if adb.exe exists`?

Reply:
I can make it optional, good point.  It's only 7 MB but I wasn't really thinking about the local build experience especially if someone had slow internet :)